app.py

```diff
@@ -23,10 +24,17 @@
 def static_file(path):
     return app.send_static_file(path)
 
+USE_RBAC = False
+
+if os.environ.get("AUTH_TYPE") == 'rbac':
+    USE_RBAC = True
```

Comment:
@pamelafox to comment on use of global here.

Reply:
This code still uses an architecture that we try to avoid, where it stores all this in globals. It makes it hard to test and overall more brittle. The preferred approach is to use an app factory pattern. Here's an example PR that ported the other RAG app to an app factory:
https://github.com/Azure-Samples/azure-search-openai-demo/pull/495

That is a larger change so probably shouldnt be a part of this PR

Reply:
We removed the global variable definition of `USE_RBAC` and instead obtained `AUTH_TYPE` from the environment variable.